src/types.ts

```diff
@@ -77,6 +79,8 @@ export interface GeckodriverParameters {
    * @default process.env.GECKODRIVER_CACHE_DIR || os.tmpdir()
    */
   cacheDir?: string
+
+  spawnOpts?: SpawnOpt
```

Comment:
Can we have a comment above for documentation?

Reply:
how does [893750d](https://github.com/webdriverio-community/node-geckodriver/pull/552/commits/893750d1c8d7ce940cbba10a397cca6165c3e720) look?

Reply:
looks like I need to add to the README as well

Reply:
both added in ec9e561

Reply:
LGTM 👍